### ISSUE TITLE
training scheme

### ISSUE THREAD

**Issue by rania-benaissa** [sswuai/PanoGAN#2]
Hello !

I am trying to train your model on my own dataset (approximately 17000 samples) with the same training scheme that was suggested for CVUSA dataset and i was wondering in which part of the code are the learning rates of the discriminators set to /10 of the generator's one (knowing that in the scripts options, the learning rate is set once and for all for all networks) and when i digged deeper into the code i found  that you multiplied by 0.5 the global discriminator's loss computation  : 

https://github.com/sswuai/PanoGAN/blob/27ab77b1644d8fe8a0e38954abbdcdf920360176/models/panogan_model.py#LL153C1-L154C96

Is the * 0.5 a normalization factor due to the fact that there are two discriminators or is it an alternative for having different learning rates for every optimizer ? 

Thank you in advance for your answer and for sharing this amazing work !
